README.md

```diff
@@ -365,13 +365,31 @@ multiple events in to a single one, a debounce time can be optionally set:
 GPIO.add_event_detect(channel, GPIO.RISING, callback=callback_fn,
 bouncetime=200)
 ```
+The thread running in the background will be idle waiting
+for an event until timeout, which can be optionally set as
+the following. The default polling timeout is 0.2 sec.
+
+```python
+# polltime set in seconds
+GPIO.add_event_detect(channel, GPIO.RISING, callback=callback_fn,
+polltime=1)
```

Comment:
What happen when polltime time out

Reply:
It will wake up and checking the thread status. If the thread status is in the running state, it will go back to the idle state waiting for an incoming event, otherwise, the thread will exit (event detection removal).
Should I add this in the README for more information?